assets/css/global.scss

```
@@ -2,6 +2,32 @@
 /* Backend Global Stylesheet */
 /*-----------------------------------------------------------------------------------*/
 
+
+/* Sensei Notice */
+.sensei-notice {
+	@import '../shared/styles/wp-admin';
+
+	border-left-color: $primary;
+	padding: 10px 12px 10px 60px;
+	background: url('../css/images/sensei-lms-icon.svg') #fff no-repeat -46px 0;
+	background-size: 35px;
+	background-origin: content-box;
+
+	p {
+		padding: 0;
+		font-size: 14px;
+	}
+
+	.button {
+		float: right;
+		margin-top: -10px;
```

A bit tight around the button, we can make smaller:

```suggestion
	.button {
		float: right;
		padding: 3px 12px;
		margin-top: -7px;
```

---

Yup, looks better.